### ISSUE TITLE
Space to move after operator not working if slot is empty

### ISSUE THREAD

**Issue by neilccbrown** [k-pet-group/Strype#270]
In GitLab by @PwtKCL on Aug 1, 2024, 11:51

Consider the following code `1|+2` where | is the text caret.
Hitting space will move the caret after the operator.

However, in the following examples `|*2`, `"test"|+"a"`, `(test)|+"a"`, it won't do the same (the cursors stays where we are).

This is because if a slot is empty, we don't process space as we can't start a slot with space (unless comments).
But we should probably still move to the next slot following the operator?

**Comment by neilccbrown** [comment]
We now don't want to do this because space in an empty slot transfers focus to the shortcuts for colour picker etc.
